cmake/thisdd4hep.sh

```diff
@@ -99,22 +98,19 @@ if [ ${Geant4_DIR} ]; then
     test -r ${G4ENV_INIT} && { cd $(dirname ${G4ENV_INIT}) ; . ./$(basename ${G4ENV_INIT}) ; cd $OLDPWD ; }
     #---- if geant4 was built with external CLHEP we have to extend the dynamic search path
     if [ @GEANT4_USE_CLHEP@ ] ; then
-	dd4hep_add_library_path @CLHEP_LIBRARY_PATH@;
+        dd4hep_add_library_path @CLHEP_LIBRARY_PATH@;
     fi;
     export CLHEP_DIR=@CLHEP_INCLUDE_DIR@/../;
     export CLHEP_ROOT_DIR=@CLHEP_INCLUDE_DIR@/../;
     export CLHEP_INCLUDE_DIR=@CLHEP_INCLUDE_DIR@;
-    dd4hep_add_library_path ${G4LIB_DIR};
```

Comment:
G4LIB_DIR is already added when G4ENV_INIT is sourced?

Reply:
yes, from geant4.sh
```sh
      47 if [ -z "${LD_LIBRARY_PATH-}" ] ; then
      48   export LD_LIBRARY_PATH="`cd $geant4_envbindir/../lib64 > /dev/null ; pwd`"
      49 else
      50   export LD_LIBRARY_PATH="`cd $geant4_envbindir/../lib64 > /dev/null ; pwd`":${LD_LIBRARY_PATH}
      51 fi
      52 
      53 
      54 #-----------------------------------------------------------------------
      55 # Setup Third-party binary and library paths...
      56 # - CLHEP
      57 
      58 if [ -z "${LD_LIBRARY_PATH-}" ] ; then
      59   export LD_LIBRARY_PATH=/cvmfs/sft.cern.ch/lcg/releases/clhep/2.4.1.3-78165/x86_64-centos7-gcc9-opt/lib
      60 else
      61   export LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:/cvmfs/sft.cern.ch/lcg/releases/clhep/2.4.1.3-78165/x86_64-centos7-gcc9-opt/lib
      62 fi
      63 
      64 
      65 # - XercesC
      66 
      67 if [ -z "${LD_LIBRARY_PATH-}" ] ; then
      68   export LD_LIBRARY_PATH=/cvmfs/sft.cern.ch/lcg/releases/XercesC/3.1.3-b3bf1/x86_64-centos7-gcc9-opt/lib
      69 else
      70   export LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:/cvmfs/sft.cern.ch/lcg/releases/XercesC/3.1.3-b3bf1/x86_64-centos7-gcc9-opt/lib
      71 fi
```